_sass/landing-page.scss

```diff
@@ -106,6 +106,35 @@
   }
 }
 
+.adopter-section {
+  max-width: 1140px;
+  margin: auto;
+  padding: 0 0 20px 0;
+  text-align: center;
+
+  .adopter-title, .adopter-description {
+    color: dimgray;
+    font-weight: 600;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
```

Comment:
```suggestion
```

Reply:
This suggestion has been implemented, correct, @jlcastrogro?

Reply:
@leecalcote Do I take these suggestions?

Reply:
@leecalcote I like more how looks it here, keeping gray color and letters uppercase
![Screenshot_20201105_131101](https://user-images.githubusercontent.com/20060947/98285933-dc6b2000-1f68-11eb-8345-64e9644e5f53.png)